static/less/shared-components.less

```diff
@@ -473,6 +473,7 @@ table.table.key-value {
   margin: 4px 0;
   font-size: 13px;
   height: 24px;
+  width: 24px;
```

Comment:
Our toasts and probably other things are broken because .loading-indicator inside .loading is absolutely positioned, which means that unless we set width, our toasts look like this.
![CleanShot 2025-04-15 at 11 21 16@2x](https://github.com/user-attachments/assets/11759098-a6b9-424c-b102-ebcefe6374e1)